module/importer-ergonode-1/src/Infrastructure/Processor/ErgonodeImportProcess.php

```diff
@@ -26,14 +26,11 @@ class ErgonodeImportProcess implements SourceImportProcessorInterface, LoggerAwa
 
     private ErgonodeZipExtractor $extractor;
     private SourceRepositoryInterface $repository;
-    /**
-     * @var ErgonodeProcessorStepInterface[]
-     */
-    private array $steps;
+    private iterable $steps;
```

Comment:
```suggestion
    /**
     * @var ErgonodeProcessorStepInterface[]
     */
    private iterable $steps;
```
Please leave the doc. Way easier to interpret the type further on